Source/PGDebugViewController.swift

```diff
@@ -62,9 +63,27 @@ public class PGDebugViewController: UIViewController {
         view.addSubview(self.tableView)
         tableView.delegate = self
         tableView.dataSource = self
-        if cellModules.count == 0 { loadFromPlistFile() }
+        if cellModules.count == 0 {
+            cellModules = loadFromPlistFile()
+            DispatchQueue.main.async {
+                self.tableView.reloadData()
+            }
+        }
         attachRefreshControlIfNeeded()
     }
+    public func getCountry() -> String?{
+        let plistData = self.loadFromPlistFile()
+        for (_, dictObject) in plistData.enumerated() {
+            let data = dictObject as? PGDictionary
+            if data?.key == "AppConfig"{
+                guard let countryString = data?.value[5] as? PGString, countryString.value.count == 2 else{
```

Comment:
Can you write a comment over this line about why you have used `data?.value[5]`?